File: JAVASCRIPT/1012.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var [A, B, C] = lines[0].split(' ').map(parseFloat);
const CONST_PI = 3.14159;

var var_Triangulo = (A * C) / 2;
var var_Circulo = CONST_PI * Math.pow(C, 2);
var var_Trapezio = ((A + B) * C) / 2;
var var_Quadrado = Math.pow(B, 2);
var var_Retangulo = A * B;

console.log("TRIANGULO: " + var_Triangulo.toFixed(3));
console.log("CIRCULO: " + var_Circulo.toFixed(3));
console.log("TRAPEZIO: " + var_Trapezio.toFixed(3));
console.log("QUADRADO: " + var_Quadrado.toFixed(3));
console.log("RETANGULO: " + var_Retangulo.toFixed(3));

File: C#/1012.cs
using System;

class Program
{
    static void Main()
    {
        string[] valores = Console.ReadLine().Split();
        double a = double.Parse(valores[0]);
        double b = double.Parse(valores[1]);
        double c = double.Parse(valores[2]);

        double areaTriangulo = (a * c) / 2;
        double areaCirculo = 3.14159 * Math.Pow(c, 2);
        double areaTrapezio = ((a + b) * c) / 2;
        double areaQuadrado = Math.Pow(b, 2);
        double areaRetangulo = a * b;

        Console.WriteLine($"TRIANGULO: {areaTriangulo:F3}");
        Console.WriteLine($"CIRCULO: {areaCirculo:F3}");
        Console.WriteLine($"TRAPEZIO: {areaTrapezio:F3}");
        Console.WriteLine($"QUADRADO: {areaQuadrado:F3}");
        Console.WriteLine($"RETANGULO: {areaRetangulo:F3}");
    }
}